src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -44,6 +46,37 @@ public JsonSerializerOptions()
             Converters = new ConverterList(this);
         }
 
+        /// <summary>
+        /// Copies the options from a <see cref="JsonSerializerOptions"/> instance to a new instance.
+        /// </summary>
+        /// <param name="options">The <see cref="JsonSerializerOptions"/> instance to copy options from.</param>
+        public JsonSerializerOptions(JsonSerializerOptions options)
+        {
+            _dictionaryKeyPolicy = options._dictionaryKeyPolicy;
+            _jsonPropertyNamingPolicy = options._jsonPropertyNamingPolicy;
+            _readCommentHandling = options._readCommentHandling;
+            _referenceHandling = options._referenceHandling;
+            _encoder = options._encoder;
+
+            _defaultBufferSize = options._defaultBufferSize;
+            _maxDepth = options._maxDepth;
+            _allowTrailingCommas = options._allowTrailingCommas;
+            _ignoreNullValues = options._ignoreNullValues;
+            _ignoreReadOnlyProperties = options._ignoreReadOnlyProperties;
+            _propertyNameCaseInsensitive = options._propertyNameCaseInsensitive;
+            _writeIndented = options._writeIndented;
+
+            Converters = new ConverterList(this);
+            for (int i = 0; i < options.Converters.Count; i++)
+            {
+                Converters.Add(options.Converters[i]);
+            }
```

Comment:
You might consider adding a copy converter to ConverterList as well:
```C#
public ConverterList(JsonSerializerOptions options, ConverterList source)
{
    _options = options;
    _list = new List<JsonConverter>(source._list);
}
```
instead of open-coding this loop.  That will correctly size the underlying list and do a bulk-copy.